clang/include/clang/Basic/DiagnosticSemaKinds.td

```diff
@@ -3696,6 +3696,12 @@ def err_sme_definition_using_sm_in_non_sme_target : Error<
   "function executed in streaming-SVE mode requires 'sme'">;
 def err_sme_definition_using_za_in_non_sme_target : Error<
   "function using ZA state requires 'sme'">;
+def err_conflicting_attributes_sme_state : Error<
```

Comment:
`arm_state` might be better than `sme_state` (as for the errors below).  The new keywords aren't in principle specific to SME.